ui-src/src/cells/holo-chat/components/channels/filterAgents.test.ts

```diff
@@ -0,0 +1,11 @@
+// import React from 'react'
+// import Chasnnels from './channels'
+// import { shallow } from 'enzyme'
+
```

Comment:
I think it is a much better solution having tests split in to their own files like this